src/controller/config.js

```diff
@@ -155,11 +165,39 @@ const drawNewImage = async (img, x, y) => {
     
 }
 
-const drawNewText = (txt, currCoordinates) => {
+const drawTextBackground = async (ctx, txt, x, y, font, padding) => {
```

Comment:
now drawing white background. Allows you to better see the text when places on the image.